org.ekstep.reviewercomments-1.0/editor/plugin.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Plugin to get comments from portal
+ * @class reviewerComments
+ * @constructor
+ * @extends org.ekstep.contenteditor.basePlugin
+ * @author Revathi P 
+ * @listens stage:render:complete
+ */
+
+org.ekstep.contenteditor.basePlugin.extend({
+    NO_COMMENTS: 'No Reviewer Comments',
+    initialize: function () {
+        ecEditor.addEventListener("stage:render:complete", this.initializeComments, this);
+    },
+    initializeComments: function (event, callback) {
+        var instance = this;
+        if (instance.comments == undefined) {
```

Comment:
if(instance.comments) is enough

Reply:
The api call is based on if comments is undefined, so if i skip the undefined check it will move on to else path to display the comments without fetch the data. So it is not possible to avoid the check